avaframe/com2AB/com2AB.py

```diff
@@ -245,6 +246,33 @@ def findSplitPoint(AvaProfile, splitPoint):
     return SplitPoint, splitPoint
 
 
+
+def readABinputs(cfgINPATH):
+    cfgAva = cfgINPATH['AvaLancheRoot']
+    cfgPath = {}
+
+    ProfileLayer = glob.glob(cfgAva + '/Inputs/LINES/*.shp')
+    cfgPath['ProfileLayer'] = ''.join(ProfileLayer)
+
+    DGMSource = glob.glob(cfgAva + '/Inputs/*.asc')
+    cfgPath['DGMSource'] = ''.join(DGMSource)
+
+    SplitPointSource = glob.glob(cfgAva + '/Inputs/POINTS/*.shp')
+    cfgPath['SplitPointSource'] = ''.join(SplitPointSource)
+
+    saveOutPath = os.path.join(cfgAva, 'Outputs/')
+    if not os.path.exists(saveOutPath):
+        # log.info('Creating output folder %s', saveOutPath)
+        os.makedirs(saveOutPath)
+    cfgPath['saveOutPath'] = saveOutPath
+
+    DefaltName = str(cfgAva).split('/')[-1]
+    cfgPath['DefaltName'] = DefaltName

```

Comment:
Not a functional issue, just correction: I'd change all to defaultName  (with the u)